decidim-proposals/app/commands/decidim/proposals/admin/create_proposal.rb

```diff
@@ -5,7 +5,7 @@ module Proposals
     module Admin
       # A command with all the business logic when a user creates a new proposal.
       class CreateProposal < Rectify::Command
-        include AttachmentMethods
+        include Decidim::AttachmentMethods
```

Comment:
In modules it is typical to have problems with namespacing. If we use the absolute namespace this will avoid having to perform many overrides
```suggestion
        include ::Decidim::AttachmentMethods
```

Reply:
The reason is that this is not widely used in `Decidim`, but yes I'll change it to use the absolute namespace.